pkg/kubectl/cmd/apply/apply_set_last_applied.go

```diff
@@ -151,8 +157,9 @@ func (o *SetLastAppliedOptions) Complete(f cmdutil.Factory, cmd *cobra.Command)
 func (o *SetLastAppliedOptions) Validate() error {
 	r := o.builder.
 		Unstructured().
+		TenantParam(o.tenant).DefaultTenant().
```

Comment:
does it need to consider both the normal form, and short-path form supporting the tenant related verbs?

Reply:
The short-path will be taken care in the apiserver. To use the tenant from the command line or the user context will be taken care by f.ToRawKubeConfigLoader(). So we are good here.